e2e/test/scenarios/binning/reproductions/23851-drill-temporal-extraction.cy.spec.js

```diff
@@ -48,12 +48,12 @@ describe("issue 23851", () => {
 
     cy.findByTestId("filter-pill").should(
       "have.text",
-      "Created At: Day of week is equal to 6",
+      "Created At: Day of week is on Friday",
     );
     cy.get("[data-testid=cell-data]").should("contain", "109.22");
     cy.icon("notebook").click();
     getNotebookStep("filter")
-      .findByText("Created At: Day of week is equal to 6")
+      .findByText("Created At: Day of week is on Friday")
```

Comment:
I think more appropriate phrasing would be "Day of week is Friday". I'll check for the ways how to change that.

Reply:
Resolved by modification to `lib.metadata.calculation/display-name-method ::varargs`